routing.rst

```diff
@@ -12,6 +12,21 @@ URL of a page from ``/blog`` to ``/news``? How many links would you need to
 hunt down and update to make the change? If you're using Symfony's router,
 the change is simple.
 
+If you want to use the annotations, be sure that you have installed the ``annotations`` recipe :
+
+.. code-block:: terminal
+
+    $ composer require annotations
+
+ To turn on the routing based on annotations, un-comment the following lines of your config/routes.yaml file by adding the ``#`` character :
```

Comment:
The first space should be removed and please add a line break after the first word crossing the 72th character (so it reads nicely in the editor).